frontend/src/components/widgets/DataFrame/DataFrame.test.tsx

```diff
@@ -98,79 +92,4 @@ describe("DataFrame widget", () => {
     expect(dataFrameContainer.size.width).toBe(400)
     expect(dataFrameContainer.size.height).toBe(400)
   })
-
-  it("Test column resizing function.", () => {
```

Comment:
Should we be deleting these tests?

Reply:
all those tests have moved to the hook implementations